config/clients/java/template/creds-OAuth2ClientTest.java.mustache

```diff
@@ -35,15 +36,16 @@ class OAuth2ClientTest {
                 Arguments.of("https://issuer.fga.example:8080/", "https://issuer.fga.example:8080/oauth/token"),
                 Arguments.of("issuer.fga.example/some_endpoint", "https://issuer.fga.example/some_endpoint"),
                 Arguments.of("https://issuer.fga.example/some_endpoint", "https://issuer.fga.example/some_endpoint"),
-                Arguments.of("https://issuer.fga.example:8080/some_endpoint", "https://issuer.fga.example:8080/some_endpoint")
-        );
+                Arguments.of(
```

Comment:
non blocking as we only did formatting change..but curious why do we have duplicate https://issuer.fga.example:8080/some_endpoint

Reply:
Is there a duplicate? I don't see it 👀

Reply:
These are the function arguments, meaning the first argument should equal the last argument after applying.

The test being called with `Arguments.of("https://issuer.fga.example:8080/some_endpoint", "https://issuer.fga.example:8080/some_endpoint")` I think is a reasonable assertion worth keeping that no information is lost (scheme, host, port, path)